src/assets/i18n/cs.json5

```diff
@@ -7286,8 +7283,8 @@
   // "search.filters.withdrawn.false": "No",
   "search.filters.withdrawn.false": "Ne",
 
-  // "search.filters.head": "Filters",
-  "search.filters.head": "Filtry",
+  // "search.filters.head": "Limit your search",
+  "search.filters.head": "Zúžit hledání",
```

Comment:
are you sure about translation of this english version? isnt better Omezit vyhledávání?